tests/hardware/tabor_tests.py

```diff
@@ -51,6 +49,4 @@ def test_config_guard(self):
         with self.assertRaises(RuntimeError):
             channel_pair.guarded_method(1, True)
 
-        self.assertFalse(channel_pair.is_in_config_mode)
-
-
+        self.assertFalse(channel_pair.is_in_config_mode)
```

Comment:
Please remove changes that do not change anything to make `git blame` more useful.

Reply:
Do you mean I should undo things like the changed formatting? 
And should I minimize merge conflicts in this file only?

Reply:
> Do you mean I should undo things like the changed formatting?

I this case yes. If we squash the commits before merging the file will not be touched.

> And should I minimize merge conflicts in this file only?

It is always a payoff.

Reply:
The unnecessary merge conflicts in tabor_tests.py should now be removed

Reply:
I added newlines to the end of the file, so it matches the one in the master branch.
But I do not think that the changes can simply remove.  In the first commit with this file, the spaces were already not there. I guess PyCharm already removed them automatically when adding the file.